frontend/src/views/SettingsView.vue

```diff
@@ -201,6 +203,28 @@ export default {
           return 'Inactive'
       }
     },
+
+
+    /**
+     * Sort users by: account management role 1st, expense authority 2nd, last name 3rd
+     */
+    sort(users) {
```

Comment:
Can we be more specific with this name? E.g. sortUsers(). In theory we could have multiple sort methods.

Reply:
Fixed.